src/components/passphrase/steps.js

```diff
@@ -0,0 +1,45 @@
+export default context => ({
+  info: {
+    cancelButton: {
+      title: 'cancel',
+      onClick: () => { context.props.closeDialog(); },
+    },
+    confirmButton: {
+      title: 'next',
+      onClick: () => { context.setState({ currentStep: 'generate' }); },
+    },
+  },
+  generate: {
+    cancelButton: {
+      title: 'cancel',
+      onClick: () => { context.props.closeDialog(); },
+    },
+    confirmButton: {
```

Comment:
the disabled 'Yes! It\'s safe' button looks weird in the `generate` step. There used to be no buttons in this step in Angular. That should be not too difficult to do in here, too.